app/assets/javascripts/host_edit.js

```diff
@@ -1,5 +1,12 @@
 //= require parameter_override
 
+var HOST_EDIT_ATTRIBUTES = {
+  "architecture" : ['architecture_id', 'organization_id', 'location_id'],
```

Comment:
@ohadlevy 
`window.tfm.globals.host_edit_architectures_override = ['architecture_id', 'organization_id']` 
for instance